.github/workflows/extract-translations.yml

```diff
@@ -37,11 +37,9 @@ jobs:
           rm catalog/autoupgrade/translations/ModulesAutoupgradeAdmin.en.xlf
           mv catalog/autoupgrade/translations/en-US/messages.en-US.xlf catalog/autoupgrade/translations/ModulesAutoupgradeAdmin.en.xlf
           if [ -d "catalog/autoupgrade/translations/en-US" ] && [ "$(ls -A catalog/autoupgrade/translations/en-US)" ]; then
-            echo "::warning::The directory catalog/autoupgrade/translations/en-US is not empty. Potential loss of translations."
-            rm -rf catalog/autoupgrade/translations/en-US
-          else
-            rmdir catalog/autoupgrade/translations/en-US
+            echo "::error::The directory catalog/autoupgrade/translations/en-US is not empty. Potential loss of translations."
```

Comment:
```suggestion
            echo "::error::The directory catalog/autoupgrade/translations/en-US is not empty, some translations are left behind. This can happen when translations have a domain set when calling the method trans."
```